src/tallies/tally.cpp

```diff
@@ -250,6 +254,42 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), 1001) !=
+                std::end(scores_);
+  if (exists == 1) {
```

Comment:
We should probably just remove this eventually, but you should be doing `if (exists)`. It's redundant as-is.